src/3d/qgs3dutils.cpp

```diff
@@ -99,7 +99,6 @@ QImage Qgs3DUtils::captureSceneImage( QgsAbstract3DEngine &engine, Qgs3DMapScene
   auto requestImageFcn = [&engine, scene] {
     if ( scene->sceneState() == Qgs3DMapScene::Ready )
     {
-      engine.renderSettings()->setRenderPolicy( Qt3DRender::QRenderSettings::RenderPolicy::OnDemand );
```

Comment:
why remove this line?

Reply:
For what I have tested, I does not change anything, it does not seem to make sense according to what we asked earlier (line 87). 

But I should move this commit to a specific PR.

Reply:
yeah let's move this to a separate PR please...